scripts/template-rules/default.js

```diff
@@ -59,8 +43,6 @@ replacements = {
 	APP_LICENSE: 'Private',
 	APP_REPO: '',
 	APP_CLIENTHOST_EXPOSE: '',
-	MAGE_REPO: MAGE_REPO,
-	MAGE_VERSION: MAGE_VERSION,
```

Comment:
Were these not part of any template?

Reply:
Not that I could find.

Reply:
I can't find them either. I think they were once used in package.json, before we open sourced it.